### ISSUE TITLE
Pagination `aria-label="results"` does not inform the user of the purpose of the region

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#3684]
This issue is from May 2023 external accessibility audit report.

**Usability issue**

**Issue ID:**
DAC_Usability_Pagination_02

**URLs:**
[https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/search-three-pages](https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/search-three-pages)

## Screen shots
<img width="360" alt="Pagination component options" src="https://github.com/alphagov/govuk-frontend/assets/56260216/f569b55f-e844-451f-b6ac-dc01468b7f3e">

The pagination region has been marked up within a `<nav>` element. However, this `<nav>` has been provided with an accessible name of ‘results’. This does not inform the user of the purpose of the region and does not convey that it contains the pagination options. As screen reader users will often navigate via regions, it is important that they are labelled adequately.

## Current code ref(s)
`#main-content > form > div.govuk-grid-row > div.govuk-grid-column-two-thirds > nav`

## Solution
All labels should be descriptive and inform users of screen reading assistive technologies of the purpose of the region, allowing them to easily discern what content is contained within the said region. Consider changing the value of the ‘aria-label’ to ‘Results pagination’.

**Comment by querkmachine** [comment]
I'm happy with that change to the default.

In the case of the block links (the stacked previous and next links), I believe the expected case here is for related content that has been split over multiple pages, rather than a list of results. Is there a reason we shouldn't use a more generic 'Pagination' label to cover both bases?
